package.json

```diff
@@ -68,7 +68,7 @@
   "peerDependencies": {
     "react": "^16.3.0",
     "react-dom": "^16.3.0",
-    "react-helmet": "^3.0.0 || ^4.0.0 || ^5.0.0"
+    "react-helmet": "^5.0.0 || ^6.0.0"
```

Comment:
```suggestion
    "react-helmet": ">=^5.0.0"
```

Reply:
How do we know this will work with `react-helmet` 7?

Reply:
We update to `>=6` or `>=7`? either way you need to update.

Reply:
I mean, how do we know the existing `seek-style-guide` code will work with a future version of `react-helmet`? Listing major versions indicates which versions of `react-table` we‘re known to work with.